pandas/core/arrays/datetimes.py

```diff
@@ -84,8 +84,6 @@
     from pandas import DataFrame
     from pandas.core.arrays import PeriodArray
 
-_midnight = time(0, 0)
```

Comment:
We're certain this is no longer used?

Reply:
Yeah I didn't find a reference to this variable anymore

```bash
% grep -R --include="*.py" _midnight pandas
pandas/io/formats/format.py:    not_midnight = values_int % one_day_nanos != 0  # type: ignore[operator]
pandas/io/formats/format.py:    both = np.logical_and(consider_values, not_midnight)  # type: ignore[arg-type]
pandas/tests/resample/test_datetime_index.py:def test_downsample_dst_at_midnight(unit):
pandas/tests/frame/methods/test_at_time.py:    def test_at_time_midnight(self, frame_or_series):
pandas/tests/indexes/datetimes/methods/test_shift.py:    def test_dti_shift_near_midnight(self, shift, result_time):
```